app/views/CreditAndRefunds.scala.html

```diff
@@ -43,170 +43,190 @@
+appConfig: config.FrontendAppConfig,
+h1: h1,
+h2: h2,
+p: p,
+link: link,
+govukInsetText : GovukInsetText
 )
 
 @(creditCharges: List[(DocumentDetailWithDueDate, FinancialDetail)], balance: Option[BalanceDetails],
-        isAgent: Boolean = false, backUrl: String, isMFACreditsAndDebitsEnabled: Boolean = false)(implicit request: Request[_], user: MtdItUser[_], messages: Messages)
+isAgent: Boolean = false, backUrl: String, isMFACreditsAndDebitsEnabled: Boolean = false)(implicit request: Request[_], user: MtdItUser[_], messages: Messages)
 @import implicitDateFormatter.longDate
 
 @getControllerHref(credit: DocumentDetail, isAgent: Boolean) = @{
-    if(isAgent) {
-        controllers.routes.PaymentAllocationsController.viewPaymentAllocationAgent(credit.transactionId)
-    } else {
-        controllers.routes.PaymentAllocationsController.viewPaymentAllocation(credit.transactionId)
-    }
+if(isAgent) {
+controllers.routes.PaymentAllocationsController.viewPaymentAllocationAgent(credit.transactionId)
+} else {
+controllers.routes.PaymentAllocationsController.viewPaymentAllocation(credit.transactionId)
+}
 }
 
 @getCreditsSummaryControllerHref(credit: DocumentDetail, isAgent: Boolean) = @{
-    if(isAgent) {
-        controllers.routes.CreditsSummaryController.showAgentCreditsSummary(credit.documentDate.getYear)
-    } else {
-        controllers.routes.CreditsSummaryController.showCreditsSummary(credit.documentDate.getYear)
-    }
+if(isAgent) {
+controllers.routes.CreditsSummaryController.showAgentCreditsSummary(credit.documentDate.getYear)
+} else {
+controllers.routes.CreditsSummaryController.showCreditsSummary(credit.documentDate.getYear)
+}
 }
 
 @getWhatYouOwe(isAgent: Boolean) = @{
-    if(isAgent) {
-        controllers.routes.WhatYouOweController.showAgent().url
-    } else {
-        controllers.routes.WhatYouOweController.show().url
-    }
+if(isAgent) {
+controllers.routes.WhatYouOweController.showAgent().url
+} else {
+controllers.routes.WhatYouOweController.show().url
+}
 }
 
 @mainTemplate(
-    pageTitle = messages("credit-and-refund.heading"),
-    backUrl = Some(backUrl),
-    isAgent = isAgent,
-    useFallbackBackLink = true
+pageTitle = messages("credit-and-refund.heading"),
+backUrl = Some(backUrl),
+isAgent = isAgent,
+useFallbackBackLink = true
 ) {
 
 
-    @h1(msg = "credit-and-refund.heading")
+@h1(msg = "credit-and-refund.heading")
 
-    @if(balance.isDefined){
-        @if(balance.get.availableCredit.get != 0){
-            <h2 class="govuk-heading-m">@{balance.get.availableCredit.get.toCurrency}
-                <span class ="govuk-body"> @messages("credit-and-refund.subHeading.has-credits")</span>
-            </h2>
-        } else {
-            <h2 class="govuk-heading-m">@{balance.get.firstPendingAmountRequested.get.toCurrency}
-                <span class ="govuk-body"> @messages("credit-and-refund.subHeading.has-no-credits")</span>
-            </h2>
-        }
+@if(balance.isDefined){
```

Comment:
if you can format code between lines: 96 - 106,

Reply:
I reformatted whole file